transformer_engine/jax/layernorm.py

```diff
@@ -202,9 +199,9 @@ def _layernorm_fp8_dot_fwd_rule(
 
     assert x.shape == ln_out.shape
 
-    kernel_amax = amax[gemm_kernel_idx, 0:1]
-    kernel_scale = scale[gemm_kernel_idx]
-    kernel_scale_inv = scale_inv[gemm_kernel_idx]
+    kernel_amax = amax_list[FP8MetaPackage.WEIGHT_IDX][0:1]
```

Comment:
Doesn't `[0:1]` equal to `[0]`?

Reply:
This is shape issue, if we do `[0]`, then the tensor shape would be `[]`, which is not compatiable to `cast` related custom calls.